styles/components/avatar.css

```diff
@@ -0,0 +1,26 @@
+.avatar {
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  margin: 0 0.7rem;
+}
+.avatar.xs {
+  width: 24px;
+  height: 24px;
+}
+.avatar.sm {
+  width: 32px;
+  height: 32px;
+}
+.avatar.md {
+  width: 40px;
+  height: 40px;
+}
+.avatar.lg {
+  width: 48px;
+  height: 48px;
+}
+.avatar.xl {
+  width: 56px;
+  height: 56px;
+}
```

Comment:
Hey I would like to say instead of using these selectors sm med lrg You can use avatar-med , avatar-sm.
This look great for readability and will help you later on with any accessibility issues too.

Reply:
Comment- UI looks really cool . This color is amazing and choice for typography is good too. And also one more thing avoid using descendant selectors as much as you can This will help when you will merge all your branches. and Its good to go :)

Reply:
done with the changes